multiserver/js_eslint.json

```diff
@@ -0,0 +1,15 @@
+{
+  "completion": "javascript",
+  "completion_item_resolve": "javascript",
+  "diagnostics": ["vscode-eslint-language-server", "javascript"],
+  "code_action": ["vscode-eslint-language-server", "javascript"],
+  "execute_command": ["vscode-eslint-language-server", "javascript"],
```

Comment:
I'm not sure if these are 100% correct, but I can't test yet since the eslint server process is starting OK but not working yet.

Reply:
syntax is correct, we should make sure vscode-eslint-language-server single server works first.

maybe missing some setting options in json file of  vscode-eslint-language-server